reference/extensions/custom_generators.rst

```diff
@@ -98,3 +98,47 @@ argument:
 .. code-block:: bash
 
     conan install --requires=zlib/1.2.13 -g MyGenerator
+
+
+Generators from tool_requires
```

Comment:
Maybe we also want to document that this can be used to inject built-in generators too?

Reply:
Does the note in https://github.com/conan-io/docs/pull/3880/files#diff-4fe044d0a3df1239f28bea0a55f5277b3f0cc08f8f17e887fc9b69c07571f449R141 work?

Reply:
Yes, sounds good